src/core/server/graph/resolvers/Subscription/commentEnteredModerationQueue.ts

```diff
@@ -38,6 +43,22 @@ export const commentEnteredModerationQueue: SubscriptionToCommentEnteredModerati
         return false;
       }
 
+      // If we're filtering by siteID, then only send back comments from the
+      // specific site.
+      if (siteID && source.siteID !== siteID) {
+        return false;
+      }
+
+      // If we're filtering by section, then only send back comments from the
+      // specific section.
+      if (
+        section &&
+        section.name !== source.section &&
```

Comment:
What about when section.name === null? I think source.section is undefined but never null